.gitignore

```diff
@@ -217,3 +217,4 @@ empty_test_environment_variables.py
 /cloud_governance/main/.test_env
 /cloud_governance/policy/send_mail.py
 cloudsensei/.env.txt
+.vscode
```

Comment:
Though harmless - do we need this :)

Reply:
We don't need to store vscode files in GitHub.

Reply:
/lgtm